content/en/docs/plugins/jackson-plugin/jackson-object-arbitrary-introspector.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "JacksonObjectArbitraryIntrospector"
+images: []
+menu:
+docs:
+parent: "jackson-plugin"
+identifier: "jackson-object-arbitrary-introspector"
+weight: 20
+---
+
+## JacksonObjectArbitraryIntrospector
+The `JacksonObjectArbitraryIntrospector` becomes the default introspector when the Jackson plugin is added.
+It puts the created properties of the given class into a map and deserializes them using Jackson's object mapper.
+
+**Example Java Class :**
+```java
+@Value // lombok getter, setter
```

Comment:
I think `setter` comment should be removed as `@Value` annotation does not generates setter.